src/Microsoft.AspNetCore.Http.Abstractions/Extensions/HeaderDictionaryExtensions.cs

```diff
@@ -11,6 +11,7 @@ public static class HeaderDictionaryExtensions
         /// <summary>
         /// Add new values. Each item remains a separate array entry.
         /// </summary>
+        /// <param name="headers">The <see cref="IHeaderDictionary"/> that this method extends.</param>
```

Comment:
Adding/modifying a dictionary or whatever isn't "extending" it, though. It's just adding/modifying stuff in it...